docs/04-GeneratingCertificates.md

```diff
@@ -44,7 +44,14 @@ Valid TLS certificates will have "Version 3" near the top of the certificate pla
 
 6. Test Steward with your certificate
 
-You can use the test client certificates included in this repo to simulate a client connection to Steward. You'll need to create a TOML config file with the `[server]` section containing the paths to your generated artifacts. See the configuration reference for more info. Your file will need to look something like this:
+A temporary test client certificate can be used verify your steward process is connectable from the network.  The test certificate MUST BE REMOVED prior to running in production.
+
+The test certificate is included in this repo at:
```

Comment:
Likewise, "The CA"

Reply:
Edited to say "test Client CA" for consistency with the above